webdav_storage/src/WebDav.php

```diff
@@ -133,7 +134,7 @@ protected function hasItems(string $uri): bool
 	 */
 	protected function mkcol(string $uri): bool
 	{
-		return $this->client->request('mkcol', $uri, [HttpClientOptions::AUTH => $this->authOptions])
+		return $this->client->request('mkcol', $uri, HttpClientAccept::DEFAULT, [HttpClientOptions::AUTH => $this->authOptions])
```

Comment:
AFAIK the `mkcol` method isn't requesting data, so we shouldn't use that parameter there.